### DISCUSSION TITLE
SMTP setup help

### DISCUSSION THREAD

**Discussion by sahilsoni0112** [lightdash/lightdash#10538] [Q&A]
Hello, 

We have a lightdash instance up and running in K8S cluster (self hosted v0.1012.0). We have also enabled chrome-browserless service (pods up & running) in our infra to support sending charts via scheduled delivery/Email. We have added below configs in order to configure SMTP email client:
(as per https://docs.lightdash.com/self-host/customize-deployment/configure-smtp-for-lightdash-email-notifications)
  EMAIL_SMTP_HOST: 
  EMAIL_SMTP_USER: 
  EMAIL_SMTP_PASSWORD: 
  EMAIL_SMTP_SENDER_EMAIL: 

Even after setting these, we dont see the email option enabled in the UI when we try to set up a scheduled delivery:
![image](https://github.com/lightdash/lightdash/assets/148484486/3c648841-043d-45aa-be88-8d06803857d2)

My question is, apart from setting the above SMTP configs, are there any extra implementation steps that we are missing in order to set up email feature? 
Any help/guidance is appreciated.

**Comment by magnew** [comment]
@sahilsoni0112, this was solved over slack, correct?

**Reply by sahilsoni0112** [reply]
correct
